resteasy-links/src/test/java/org/jboss/resteasy/links/test/TestLinks.java

```diff
@@ -90,13 +92,14 @@ public void after(){
    @Test
    public void testLinks() throws Exception
    {
-      Book book = client.getBookXML("foo");
-      checkBookLinks1(url, book);
-      book = client.getBookJSON("foo");
-      checkBookLinks1(url, book);
+//      Book book = client.getBookXML("foo");
+//      checkBookLinks1(url, book);
+      Book book = client.getBookJSON("foo");
+//      checkBookLinks1(url, book);

```

Comment:
Considering the test is now ignored, please leave the lines untouched here unless they hurt anything else